packages/sanes-chrome-extension/src/routes/account/index.tsx

```diff
@@ -44,10 +46,12 @@ const AccountView = (): JSX.Element => {
 
   const items = [
     {
+      icon: recoveryPhrase,
       text: 'Show recovery words',
```

Comment:
Maybe `Show mnemonic`? I do not like the two lines effect...

Reply:
We understand `Show mnemonic` but it may confuse non-tech users. To cut down on the text used I suggest `Recovery words`